alfredo-zimperz/.scannerwork/css-bundle/bin/server

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env node
+
+/*
+* This script expects following arguments
+*
+* port - port number on which server should listen
+* host - host address on which server should listen
+*/
+
```

Comment:
lo que comentamos ayer de los espacios tras for, function, if, else, etc.... y recuerda primero constantes y despues variables